code/game/objects/items/stacks/medical.dm

```diff
@@ -134,7 +134,10 @@
 		return heal_carbon(patient, user, heal_brute, heal_burn)
 	patient.balloon_alert(user, "can't heal that!")
 
-/obj/item/stack/medical/proc/try_heal_checks(mob/living/carbon/patient, mob/user, brute, burn)
+/obj/item/stack/medical/proc/try_heal_checks(mob/living/carbon/patient, mob/user, brute, burn, looping = FALSE)
+	if(looping)
+		balloon_alert(user, "assessing damage...")
+		do_after(user, 1 SECONDS, patient)
```

Comment:
You're not actually early returning here with ``return FALSE``, but I feel like this isn't what you want it to be doing? Every attempt at healing will give you the balloon alert and an extra second waiting, which is a little spam-ey and even longer now.

Reply:
@JohnFulpWillard should be fine now i think
also this balloon alert only shows up when the healing loops, i.e., after the 1st heal completes, so that you know to switch limb

Reply:
Trying it and it does look a lot more spammy than previously, you need to go through 2 do-afters for each heal which feels a lot worse

https://github.com/tgstation/tgstation/assets/53777086/63223efe-ab02-4569-8fe8-4e5244e04446

Reply:
@JohnFulpWillard the total time is unchanged

the extra do after only appears after the first heal, its to give you time to change zone selected

i can increase the time of the 1st do_after to 1.5s if you want? and obviously adjust the 2nd do_after by subtracting 0.5s